tcf_website/templates/course/course_professor.html

```diff
@@ -21,10 +21,30 @@
                     <h1 class="mr-md-3 text-nowrap">{{ course.code }}</h1>
                     <h2 class="mr-md-3">{{ course.title }}</h2>
                 </div>
-                <h3 class="font-weight-light text-tcf-orange">
-                  <a href="{% url 'instructor' instructor_id=instructor.id %}">
-                    {{ instructor.full_name }}</a>
-                </h3>
+
+                <div class="d-flex flex-row">
+                  <!-- Instructor Dropdown -->
+                  <div class="dropdown btn-group">
+                    <a role="button" class="h3 font-weight-light text-tcf-orange mb-0"
+                       href="{% url 'instructor' instructor_id=instructor.id %}">
+                       {{ instructor.full_name }}</h3>
+                    </a>
+                    <!-- Store course ID here for jQuery -->
+                    <a role="button" class="btn dropdown-toggle dropdown-toggle-split mb-0"
+                       data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                      <span class="sr-only">Toggle Dropdown</span>
+                    </a>
+                    <div id="instructor" class="dropdown-menu dropdown-menu-right" disabled>
+                      <!-- To be populated by instr_data.js script -->
+                    </div>
+                  </div>
+
+                  <!-- Semester last taught -->
+                  <div class="ml-auto btn-group">
+                    <button class="btn bg-white disabled">Last taught</button>
+                    <button class="btn bg-tcf-blue text-white">{{ semester_last_taught }}</button>
```

Comment:
I like the look of this but it's a bit confusing to make these buttons
Not sure what a simple fix here would be

Reply:
yeah maybe I can make them square instead of rounded or something?

Reply:
I second this, it's especially confusing when half of the bar is highlighted after clicked

Reply:
I'll remove the styling for hover/click or just make it not a button